packit_service_fedmsg/consumer.py

```diff
@@ -78,15 +86,19 @@ def fedora_messaging_callback(self, message: Message):
         :param message: Message from Fedora message bus
         :return: None
         """
-        if message.body.get("user") != "packit":
-            logger.info("Not built by packit!")
-            return
-
         if message.topic not in INTERESTED_TOPICS:
```

Comment:
```suggestion
        if message.topic not in COPR_TOPICS | KOJI_TOPICS:
```

And then INTERESTED_TOPICS could be removed getting rid of some duplicity.

Reply:
This will be removed in #14 ..;) (I want to rebase once merged.)

Reply:
#14 has been merged